bom/models.py

```diff
@@ -690,13 +693,17 @@ def as_dict_for_export(self):
             'seller': self.seller.name,
             'unit_cost': self.unit_cost,
             'minimum_order_quantity': self.minimum_order_quantity,
-            'nre_cost': self.nre_cost
+            'nre_cost': self.nre_cost,
+            'preferred': self.preferred,
```

Comment:
Adding `preferred` and `date` to unique_together would allow two SellerParts to exist with the same seller, cost, moq, nre; so long as one is preferred and one is not, for example. Or similarly if the date property is different. Is this intentional? If so, can you explain the logic?

Reply:
Indeed, at least `preferred` should be in the `unique_together` list. About the date, this can be discussed.

The idea behind this implementation is that, due to the component shortage, I don't necessarily buy the components from the cheapest supplier, mainly because they're not available. Also, I want to keep a history of component prices, but also be able to indicate which is preferred (basically, which one I bought) for price calculation.

The date has been added to keep a trace of price evolutions over time. So the question is: is it realistic to encode the same seller_parts, with an already existing Seller / Cost / MOQ / NRE? Probably it is, even if I think I didn't have such a case so far, as every time I ask for an actualized quote, the prices change. 

So, I'll add `preferred` and `date` in the `unique_together` list.